appveyor.yml

```diff
@@ -29,23 +30,33 @@ install:
       }
       scripts\check-sha256.ps1 cmake-3.10.1-win64-x64.zip 8251F70C85B58F3CA1F24E4A3B0637E2D609B5E4A341D00B70E02E89244D5029
       Start-Process -FilePath '7z' -ArgumentList 'x','cmake-3.10.1-win64-x64.zip','-oC:\' -Wait
+  - ps: |
+      if (!(Test-Path ninja-win.zip)) {
+          appveyor DownloadFile https://github.com/ninja-build/ninja/releases/download/v1.8.2/ninja-win.zip
```

Comment:
Instead of downloading this, we can also commit this to the repo like we do for Linux and macOS.